java/format/pom.xml

```diff
@@ -33,6 +33,7 @@
     <dependency>
       <groupId>com.google.flatbuffers</groupId>
       <artifactId>flatbuffers-java</artifactId>
+      <version>${dep.fbs.version}</version>
```

Comment:
I think if we leave flatbuffers in the dependency management section, then this could be removed also

Reply:
yes.  done.